README.md

```diff
@@ -55,6 +55,25 @@ Images are customizable, default ones are used when no image is provided.
     NSLog(@"%0.2f - %0.2f", slider.leftValue, slider.rightValue);
 }
 
+//For Swift 3
+func rangeSliderValueDidChange(_ slider: MARKRangeSlider) {
+        print("right \(slider.rightValue) left \(slider.leftValue)")
```

Comment:
vadymmarkov is talking about spacing, line 60 ("print") seems to be one tab further than it should. Look at the line 55 ("NSLog"), both lines must be aligned.